liblangutil/ParserBase.cpp

```diff
@@ -57,30 +57,79 @@ Token ParserBase::advance()
 	return m_scanner->next();
 }
 
+std::string ParserBase::tokenName(Token _token)
```

Comment:
"tokenDescription"? I know this is existing code, but it deserves a better name now that it is not only function-local anymore.

Reply:
Good! I'll do this in the next iteration.